addons/web/static/src/views/form/form_controller.scss

```diff
@@ -547,54 +547,17 @@
         clear: both; // For the notebook not to have alongside floating elements
         margin-top: $o-form-spacing-unit * 2;
 
-        > .o_notebook_headers {
+        .o_notebook_headers {
             @include o-form-sheet-negative-margin;
```

Comment:
I wish... Sadly this set of mixins are used a couple of times... and not always with Notebook...
My proposition is to keep them for now and take advantage of the margin/padding custom properties once we tackle the other ones.

Reply:
I'm not proposing to remove the mixin, just to don't use it here since the component already provides the necessary hooks to manage the layout without overrides.
We can keep the mixin for the other elements of course